pegasus/sites.v3/code.org/views/video_player.js.haml

```diff
@@ -1,13 +1,12 @@
-%script{src: "/js/videojs-ie8.min.js"}
-%script{src: "/js/video.min.js"}
-
 %link{href: "/css/video-js.css", rel: "stylesheet"}
```

Comment:
Would it be possible to conditionally load this `video-js.css` content as well? This is another unnecessary extra download when we're not using the fallback player.